src/Schema/Events/ImpressionTracking.ts

```diff
@@ -259,3 +259,28 @@ export interface EditProfileModalViewed {
   inquiry_id: string
   platform: Platform
 }
+
+/**
+ * User sees the banner in the app
+ *
+ * This schema describes events sent to Segment from [[BannerViewed]].
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "bannerViewed",
+ *    context_screen: "home",
+ *    context_module: "paymentFailed",
+ *    item_type: "order" | "orders"
```

Comment:
This will allow us to differentiate if the banner was for one failed payment or multiple orders with failed payments. Does that make sense? Is that how we can do this? Is it important to track if people see banner for one failed payment or multiple failed payments?

Reply:
I think it's not harm, but also not super necessary. The idea is understanding more generally if the view the banner > clicked on it